tests/unit/models/reminder-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
```

Comment:
If you're going to keep a test file in the PR make sure you are testing something legit. Bare minimum you could test that the model name is a string..stuff like that.